lib/oktaClient.js

```diff
@@ -2,7 +2,7 @@ const okta = require('@okta/okta-sdk-nodejs');
 require('dotenv').config();
 
 const client = new okta.Client({
-  orgUrl: process.env.OKTA_ORG_URL,
+  orgUrl: `https://${process.env.OKTA_DOMAIN}/oauth2/default`,
```

Comment:
Why are we not using the okta config from up above? It already has the URL and other config elements. Maybe we should be using that instead of just these 2 params.

Reply:
good question